src/app/core/market/api/cart/cart.service.ts

```diff
@@ -56,12 +56,9 @@ export class CartService {
   clearCart(): Observable<any> {
     this.log.d(`Clearing cart with id=1`);
     return this.market.call('cart', ['clear', 1]).do(
-      data => {
-        this.snackbar.open('All Items Cleared From Cart')
-        this.updateCart();
-      },
+      data => this.updateCart(),
```

Comment:
could just be
`return this.market.call('cart', ['clear', 1]).do(this.updateCart, this.snackbar.open);`